src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -193,7 +193,15 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
     // Method Description:
     // - Loads the search box from the xaml UI and focuses it.
-    void TermControl::CreateSearchBoxControl()
+    // Arguments:
+    // - populateFromSelection: if this flag set to true and there is a text is selected inside terminal,
+    // we should try to use the selected text to populate the search box.
+    // - allowMultiLineSelection: if this flag set to true and multiple lines are selected,
+    // we should use all the lines to populate the search box.
+    // If the flag is set to false and multiple lines are selected,
+    // we should not use the selected text to populate the search box.
+    // - caseSensitive: boolean that represents if the current search is case sensitive

```

Comment:
```suggestion
```
I don't see a `caseSensitive` param here haha

Reply:
My bad. Will fix it (in any case need to work on docs)

Reply:
Oh yeah, I completely forgot to ask you to make a PR on the docs site haha.